.eleventy.js

```diff
@@ -28,16 +28,16 @@ module.exports = function(eleventyConfig) {
   };
   eleventyConfig.addPlugin(cacheBuster(cacheBusterOptions));
 
-  // Add CSS minifier
-  eleventyConfig.addFilter("cssmin", function(code) {
-    return new CleanCSS({}).minify(code).styles;
-  });
+  // Add CSS minifier -- replaced by a gulp task
+  // eleventyConfig.addFilter("cssmin", function(code) {
+  //   return new CleanCSS({}).minify(code).styles;
+  // });
```

Comment:
No need to comment out — OK to commit the change separately and write a message for this

Reply:
@thinkbulecount2 please resolve review comments once they are done!